ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -69,6 +70,14 @@ public partial class PrincipiaPluginAdapter : ScenarioModule {
 
   private Krakensbane krakensbane_;
   private NavBall nav_ball_;
+  private UnityEngine.Texture compass_nav_ball_texture_;
+  private UnityEngine.Texture inertial_nav_ball_texture_;
+  private UnityEngine.Texture barycentric_nav_ball_texture_;
+  private bool nav_ball_changed_ = true;
+
+  private bool override_rsas_target_ = false;
```

Comment:
This abbreviation has no known meaning, although it's probably related to SAS, whose meaning is itself ineffable; see http://wiki.kerbalspaceprogram.com/wiki/SAS. Documenting the ineffability.